netlify.toml

```diff
@@ -0,0 +1,12 @@
+# Hugo build configuration for Netlify
+# (https://gohugo.io/hosting-and-deployment/hosting-on-netlify/#configure-hugo-version-in-netlify)
+
+[build]
+command = "npm run build:preview"
+publish = "public"
+
+[build.environment]
+GO_VERSION = "1.22.2"
+
+[context.production]
+command = "npm run build:production"
```

Comment:
I might be misunderstanding, but is this config file used to build and deploy the pull request previews and the production site?

Reply:
We are currently using GitHub pages for the production site.

Reply:
Filed https://github.com/dandi/dandi-about/issues/95.